ray-operator/config/samples/ray-service-alb-ingress.yaml

```diff
@@ -1,27 +0,0 @@
-apiVersion: networking.k8s.io/v1
```

Comment:
There is no doc refers to this YAML. This is useless.

Reply:
It seems not so correct, there still exists [a link ](https://github.com/ray-project/ray/blob/c54437c42fa138580a0367f813b8c4bd9ca0b3e8/doc/source/serve/production-guide/kubernetes.md?plain=1#L243)points to the `v1.0.0` version, if we bump the link to `v1.3.0` version or later, it will be a broken link.